### ISSUE TITLE
Update TA1 Service Code to AMR extraction endpoint to allow for an additional metadata field

### ISSUE THREAD

**Issue by tangopapatime** [DARPA-ASKEM/knowledge-middleware#8]
Add a new field to the Code To AMR endpoint to accept a `metadata` object that describes which block of code needs to be used for the Code-to-AMR extraction.
This is what @YohannParis and I came up with.

```json
"metadata": {
    "dynamics": [
        {
            "name": "AMR code",
            "filename": "SIDARTHE.py",
            "block": "L205-L213",
        }
    ]
}
```

**Comment by brandomr** [comment]
This looks good to me. Is the `name` meant to be used for anything or is it just a label for the HMI for use later?

**Comment by YohannParis** [comment]
> This looks good to me. Is the `name` meant to be used for anything or is it just a label for the HMI for use later?

The `name` is for pure UX purposes, it is not required.

**Comment by brandomr** [comment]
Closing this as it will be addressed by #54
